src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -59,7 +59,16 @@
 import javax.annotation.Nullable;
 
 import it.unimi.dsi.fastutil.longs.LongSet;
+import net.minecraft.Util;
```

Comment:
Please do not reorganize imports.
Please update the license header in this file